src/main/java/com/egen/controller/OrderController.java

```diff
@@ -1,50 +1,58 @@
 package com.egen.controller;
 
+import com.egen.Service.OrderService;
 import com.egen.model.Order;
-import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
 
 import java.time.ZonedDateTime;
-import java.util.Collections;
 import java.util.List;
 
 @RestController
+@ResponseBody
+@RequestMapping("order")
 public class OrderController {
     /**
      * implement the following endpoints
      */
 
-    @GetMapping("order")
-    public ResponseEntity<List<Order>> getAllOrders(){
-        //TODO
-        return ResponseEntity.ok(Collections.singletonList(new Order("id")));
-    }
-
-    public ResponseEntity<List<Order>> getOrderById(String id){
-        //TODO
-        return null;
-    }
-
-    public ResponseEntity<List<Order>> getAllOrdersWithInInterval(ZonedDateTime startTime, ZonedDateTime endTime){
-        //TODO
-        return null;
-    }
-
-    public ResponseEntity<List<Order>> top10OrdersWithHighestDollarAmountInZip(String zip){
-        //TODO
-        return null;
-    }
-
-    public ResponseEntity<Order> placeOrder(Order order){
-        return null;
-    }
-
-    public ResponseEntity<Order> cancelOrder(Order order){
-        return null;
-    }
-
-    public ResponseEntity<Order> updateOrder(Order order){
-        return null;
-    }
-}
+        @Autowired
+        private OrderService orderService;
+
+        @GetMapping("")
+        public List<Order> getAllOrders(){
+            //TODO
+            return (List<Order>) orderService.getAllOrders();
+        }
+            @GetMapping("{id}")
+            public Order getOrderById(@PathVariable("id") String orderId){
+                return orderService.getOrderById(orderId);
+            }
+
+            @GetMapping("{start}/{end}")
+            public List<Order> getAllOrdersWithinInterval (@PathVariable("start") ZonedDateTime startTime, @PathVariable("end") ZonedDateTime endTime){
+                return orderService.getAllOrdersWithinInterval(startTime, endTime);
+        }
+
+                @GetMapping("getTopTen/{zip}")
+                public List<Order> top10OrdersWithHighestDollarAmountInZip(String zip){
+
+                        return orderService.top10OrdersWithHighestDollarAmountInZip(zip);
+                    }
+
```

Comment:
indentation needs to be fixed here. Its best to always keep the code clean and readable.